app/src/main/java/com/example/wnews/views/home/news/presenter/NewsPresenter.kt

```diff
@@ -0,0 +1,88 @@
+package com.example.wnews.views.home.news.presenter
+
+import android.util.Log
+import android.widget.Adapter
+import androidx.lifecycle.MutableLiveData
+import androidx.recyclerview.widget.RecyclerView
+import com.example.wnews.models.ListNews
+import com.example.wnews.models.News
+import com.example.wnews.models.UserAuth
+import com.example.wnews.providers.RetrofitProvider
+import com.example.wnews.views.home.news.NewsView
+import com.example.wnews.views.home.news.adapter.NewsAdapter
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class NewsPresenter(val newsView: NewsView) {
+
+    var arrayListNews = arrayListOf<News>()
+        private set
+
+    var loading = false
```

Comment:
Boolean state variables are usually prefixed with the `is` word. For example this variable would be `isLoading`